packages/fxa-auth-server/lib/senders/emails/partials/automatedEmailNoAction/en.ftl

```diff
@@ -1,4 +1,3 @@
-# supportLink - https://accounts.firefox.com/support (requires subscription to visit this URL)
-automated-email = This is an automated email; if you received it in error, no action is required.
-  For more information, please visit <a data-l10n-name="supportLink">{ -brand-mozilla } Support</a>.
-automated-email-plaintext = This is an automated email; if you received it in error, no action is required.
+# supportLink - https://support.mozilla.org/kb/im-having-problems-my-firefox-account
+automated-email-no-action = { automated-email-no-action-plaintext } For more info, visit <a data-l10n-name="supportLink">{ -brand-mozilla } Support</a>.
+automated-email-no-action-plaintext = This is an automated email. If you received it by mistake, you don’t need to do anything.
```

Comment:
While we are updating partials, can we have both MJML and TXT have the same wording and then delete the `support` partial?

Reply:
> While we are updating partials

😭 

Well the support txt partial is used with more than just the no action partial so I can't remove it even after I move that text into the no action partial.

Reply:
Ah, I am aligned - updating that would definitely be outside the scope of this ticket, as it would affect a lot of other emails. I filed a follow-up ticket to clean up the footer partials and add them to Storybook,so it can be handled then. :)